package.json

```diff
@@ -64,7 +64,6 @@
     "nyc": "^15.0.1",
     "prettier": "^2.6.2",
     "pretty-quick": "^3.1.3",
-    "rimraf": "^3.0.2",
```

Comment:
Rimraf is unnecessary here and would fail when trying to build after removing all dev dependencies.